### DISCUSSION TITLE
milliSecond should long value ?

### DISCUSSION THREAD

**Discussion by coder-free** [querydsl/querydsl#2932] [Q&A]
## Observed vs. expected behavior

![image](https://user-images.githubusercontent.com/2630290/116653237-5846a200-a9b9-11eb-8829-27e5e352542a.png)

## Steps to reproduce

None

## Environment

IntellijIDEA, java 

Querydsl version: 4.4.0

Querydsl module: types ?

Database:

JDK: 1.8

## Additional details

`DateTimePath<java.util.Date>`
```
    public NumberExpression<Integer> milliSecond() {
        if (this.milliseconds == null) {
            this.milliseconds = Expressions.numberOperation(Integer.class, DateTimeOps.MILLISECOND, new Expression[]{this.mixin});
        }

        return this.milliseconds;
    }
```
I think the milliSecond is Long value.
so this method should return `NumberExpression<Long>`

**Comment by jwgmeligmeyling** [comment]
Why? Its not the epoch millisecond timestamp, its the milliseconds portion, which is 1000 at most.
